examples/s2s-api/oxxo_payment.php

```diff
@@ -90,17 +91,24 @@
     $response = $payment->postRequest();
 
     echo "\n\nRESULT BELOW\n";
-    if ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_VERIFICATION_REQUIRED) {
+    if ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_OK) {
         echo "The transactions was successfully processed. Vendo's Transaction ID is: " . $response->getTransactionDetails()->getId();
-        echo "\nThe OXXO PDF (gzipped and base64 encoded): " . $response->getOxxoPaymentResult()->getPdfGzipBase64();
-        echo "\nThe OXXO PNG (gzipped and base64 encoded): " . $response->getOxxoPaymentResult()->getPngGzipBase64();
-        echo "\nThe expiration date (gzipped and base64 encoded): " . $response->getOxxoPaymentResult()->getExpiresAt();
-        echo "\nThe OXXO barcode: " . $response->getOxxoPaymentResult()->getBarcode();
+        echo "\nThe payment Auth Code is: " . $response->getCreditCardPaymentResult()->getAuthCode();
+        echo "\nThe Payment Details Token is: ". $response->getPaymentToken();
```

Comment:
Let's not use the Payment Token for OXXO. 
Add a line to echo the Verification ID instead.